### DISCUSSION TITLE
Separate twinbasic-lang repo?

### DISCUSSION THREAD

**Discussion by mansellan** [twinbasic/twinbasic#376] [Ideas]
Is it worth having a separate repository for discussion of twinBASIC language design?

My thinking is this: the current repository is used for bug reports, feature requests, support requests, and language proposals.

Language design is unique amongst those, as it requires in-depth consideration. Once twinBASIC reaches v1, breaking changes also need to be considered. Mistakes made in language design can be difficult or impossible to resolve once a language is released.

C# (since the .Net Core reset) have done their language design in the open, at https://github.com/dotnet/csharplang. Would it be worth opening a repo dedicated to the design and evolution of the twinBASIC language?

**Comment by WaynePhillipsEA** [comment]
I like this idea, and it would certainly help tidy up this repo.   There were some interesting proposals / discussions from over at vbforums last week that would've been perfectly placed in such a repo (although getting them to move over to github to discuss the ideas in more detail might prove challenging...)

**Comment by mansellan** [comment]
If you decide to implement this, you can transfer existing issues in the same org: https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository
